src/components/Card/Card.tsx

```diff
@@ -1,27 +1,47 @@
 import React, { useState } from 'react';
 import classNames from 'classnames';
 
-type Props = {
+type TTagVariants = 'default' | 'minor' | 'light';
+
+type TCardType =
+  | 'success'
+  | 'informational'
+  | 'error'
+  | 'attention'
+  | 'warning'
+  | 'neutral';
+
+type TImg = {
+  src: string;
+  alt: string;
+};
+
+export type TCardProps = {
   disabled?: boolean;
   error?: boolean;
-  imgSrc: string;
-  imgAlt: string;
-  tagColor?: string;
-  tagBg?: string;
+  type?: TCardType;
+  tagType?: TTagVariants;
+  tagText: string;
```

Comment:
```suggestion
  tag?: {
    type?: TTagVariants;
    text: string;
  };
```
Czemu jako 2 osobne propsy?